lib/mayaUsd/fileio/utils/meshReadUtils.cpp

```diff
@@ -615,6 +616,23 @@ void UsdMayaMeshReadUtils::assignPrimvarsToMesh(
             continue;
         }
 
+        // Exclude primvars if they match with the exclude namespace
+        std::string primVarString = fullName.GetString();
+
+        bool skipPrimvar = false;
+        for (const TfToken& primVarNameSpace : excludePrivarNamespaceSet) {
+            std::string primVarNameSpaceString = primVarNameSpace.GetString();
+
+            // The primVarString is a substring of excludePrimVarNamespace string from the beginning
+            if (primVarString.find(primVarNameSpaceString) == 0) {
```

Comment:
I guess the probability that a prim var has no namespace but has a name that matches a namespace is low? IDK if there is a way to extract just the namespace from the primVar (maybe [rimVar.GetPrimVarNamsapce or some such?)

Reply:
Yes I think I can do that

Reply:
Instead of using GetNamepsace, I used `primvar.NameContainsNamespaces()` to check if the primvar we are importing does have a namespace. We only proceed the checking if this primvar has a namespace. In this way we can avoid the corner case you mentioned and be more efficient at the mean time.